pkg/kn/commands/revision_get_test.go

```diff
@@ -82,11 +111,69 @@ func TestRevisionGetDefaultOutput(t *testing.T) {
 	testContains(t, output[2], []string{"bar", "bar-wxyz"}, "value")
 }
 
-func createMockRevisionWithParams(name, svcName string) *v1alpha1.Revision {
+func TestRevisionGetOneOutput(t *testing.T) {
+	revision := createMockRevisionWithParams("foo-abcd", "foo", true)
+	action, output, err := fakeRevisionGet([]string{"revision", "get", "foo-abcd"}, revision, true)
+	if err != nil {
+		t.Fatal(err)
+	}
+	if action == nil {
+		t.Errorf("No action")
+	} else if !action.Matches("get", "revisions") {
+		t.Errorf("Bad action %v", action)
+	}
+	testContains(t, output[0], []string{"SERVICE", "NAME", "AGE", "CONDITIONS", "READY", "REASON"}, "column header")
+	testContains(t, output[1], []string{"foo", "foo-abcd"}, "value")
+}
+
+func TestRevisionGetOutputWithTwoRevName(t *testing.T) {
+	t.Log("create a first revision name foo")
+	revision := createMockRevisionWithParams("foo-abcd", "foo", true)
+	action, output, err := fakeRevisionGet([]string{"revision", "get", "foo-abcd"}, revision, true)
+	if err != nil {
+		t.Fatal(err)
+	}
+	if action == nil {
+		t.Errorf("No action")
+	} else if !action.Matches("get", "revisions") {
+		t.Errorf("Bad action %v", action)
+	}
+	testContains(t, output[0], []string{"SERVICE", "NAME", "AGE", "CONDITIONS", "READY", "REASON"}, "column header")
+	testContains(t, output[1], []string{"foo", "foo-abcd"}, "value")
+
+	t.Log("create a second revision name bar")
+	revision = createMockRevisionWithParams("bar-abcd", "bar", true)
+	action, output, err = fakeRevisionGet([]string{"revision", "get", "bar-abcd"}, revision, true)
+	if err != nil {
+		t.Fatal(err)
+	}
+	if action == nil {
+		t.Errorf("No action")
+	} else if !action.Matches("get", "revisions") {
+		t.Errorf("Bad action %v", action)
+	}
+	testContains(t, output[0], []string{"SERVICE", "NAME", "AGE", "CONDITIONS", "READY", "REASON"}, "column header")
+	testContains(t, output[1], []string{"bar", "bar-abcd"}, "value")
+
+	t.Log("verify by passing two revision name to get command")
+	_, output, err = fakeRevisionGet([]string{"revision", "get", "foo-abcd", "bar-abcd"}, revision, true)
+	assert.NotNil(t, err)
+	assert.Empty(t, output)
+
+}
+
+func createMockRevisionWithParams(name, svcName string, version bool) *v1alpha1.Revision {
+	var apiVersion string
+	if version {
```

Comment:
Why do you have different api versions ?

Reply:
This condition i have added to check the **apiVersion**
Because in the base code
when we do 
```
kn revision get -o yaml

apiVersion: knative.dev/v1alpha1
items:
- apiVersion: serving.knative.dev/v1alpha1
  kind: Revision
  metadata:
    annotations:
      autoscaling.knative.dev/target: "10"
      serving.knative.dev/lastPinned: "1559284138"
    creationTimestamp: "2019-05-31T06:28:47Z"
    generateName: autoscale-go-
    generation: 1
```
If its a list then it is giving apiversion as "knative.dev"